commit 684713e65f4d5fa769fa4ea1997c847025e04f5d (refs/changes/34/3834/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-04-17T22:37:40+00:00 (1 year, 6 months ago)
    
    TRITON-328 Update NAPI to restify-warden@1.1.0
    Reviewed by: Nick Zivkovic <nick.zivkovic@joyent.com>
    Approved by: Nick Zivkovic <nick.zivkovic@joyent.com>

diff --git a/lib/util/validate.js b/lib/util/validate.js
index 24f8731..98d17ce 100644
--- a/lib/util/validate.js
+++ b/lib/util/validate.js
@@ -346,7 +346,7 @@ function validateVxlanID(_, name, vlan_id, callback) {
 module.exports = {
     bool: validateBoolean,
     enum: validateEnum,
-    fieldsArray: warden.fieldsArray,
+    fieldsArray: warden.arrayOfFields,
     IP: validateIP,
     IPv4: validateIPv4,
     ipArray: validateIParray,
@@ -364,10 +364,10 @@ module.exports = {
     uuidPrefix: warden.uuidPrefix,
     string: warden.string,
     stringOrArray: warden.stringOrArray,
-    stringArray: warden.stringArray,
+    stringArray: warden.arrayOfString,
     subnet: validateSubnet,
-    UUID: warden.UUID,
-    UUIDarray: warden.UUIDarray,
+    UUID: warden.uuid,
+    UUIDarray: warden.arrayOfUuid,
     VLAN: validateVLAN,
     VxLAN: validateVxlanID
 };
diff --git a/package.json b/package.json
index ea1e7b6..832ad6a 100644
--- a/package.json
+++ b/package.json
@@ -19,7 +19,7 @@
     "node-uuid": "1.4.7",
     "portolan-moray": "git+https://github.com/joyent/sdc-portolan-moray.git#7e2c4ca",
     "restify": "4.1.1",
-    "restify-warden": "0.2.0",
+    "restify-warden": "1.1.0",
     "sdc-clients": "9.2.0",
     "tape": "4.5.1",
     "trace-event": "1.3.0",
